cookbooks/bcpc-hadoop/recipes/yarn_site.rb

```diff
@@ -1,3 +1,6 @@
+node.default['bcpc']['hadoop']['yarn']['site_xml']['yarn.log-aggregation.retain-seconds'] =
```

Comment:
The default for this is -1 for never expire, while I agree that we want to clean out these files once in a while, can we also have a setting that if default['bcpc']['hadoop']['yarn']['yarn.log-aggregation.retain-days'] == -1 just bypass all the math and set to yarn.log-aggregation.retain-seconds -1?